### ISSUE TITLE
Store, list and export all of the actions in Hatstall

### ISSUE THREAD

**Issue by dicortazar** [chaoss/grimoirelab-hatstall#76]
Not sure if this should be a feature request here or in SortingHat.

The point is that it would be useful to store and read somewhere the last actions undertaken in a database.

The use case for this is that as the service sometimes crashes, as any other service, I may not be sure if my last changes where applied to the database and up to what percentage of those were applied.

If I would have the last set of changes, this is quite useful for the following reasons:
* I would be aware of the work invested in updating affiliations and identities.
* I can easily reproduce this and send a curated version of the dataset and the steps needed to other people starting an analysis of a similar ecosystem of mine with a similar set of developers.
  * Let's say that I can start analyzing GrimoireLab-Perceval and then others extend the analysis. I could give them my curated version of the database or at least the changes I had to made in Hatstall.
* As there's not a real user access so far, it would be useful to double check what others may have applied to the existing dataset.

Cheers,
Daniel.

**Comment by jsmanrique** [comment]
Perhaps latest work on some PRs to SortingHat could help on this:
chaoss/grimoirelab-sortinghat#218

**Comment by dicortazar** [comment]
Yep, that ticket may cover part of the functionality detailed here.
